internal/auth0/connection/resource_scim.go

```diff
@@ -0,0 +1,88 @@
+package connection
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/auth0/terraform-provider-auth0/internal/config"
+	internalError "github.com/auth0/terraform-provider-auth0/internal/error"
+	internalSchema "github.com/auth0/terraform-provider-auth0/internal/schema"
+)
+
+// NewSCIMConfigurationResource will return a new auth0_scim_configuration (1:1) resource.
+func NewSCIMConfigurationResource() *schema.Resource {
+	return &schema.Resource{
+		Schema:        scimResourceSchema,
+		CreateContext: createSCIMConfiguration,
+		UpdateContext: updateSCIMConfiguration,
+		ReadContext:   readSCIMConfiguration,
+		DeleteContext: deleteSCIMConfiguration,
+		Importer: &schema.ResourceImporter{
+			StateContext: internalSchema.ImportResourceGroupID("client_id"),
+		},
+		Description: "With this resource, you can use `SCIM` to manage a connection.",
```

Comment:
We should probably be a bit more verbose here. Link some docs, add more description.

Reply:
I found some docs to link